model_server_tools/model_packaging/export_model_utils.py

```diff
@@ -0,0 +1,289 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+"""
+Helper utils for Model Export tool
+"""
+
+import os
+import json
+import zipfile
+import sys
+import logging
+from model_server_tools.model_packaging.manifest_components.publisher import Publisher
```

Comment:
Why use full path?
it's better use relative path, right?

Reply:
Relative path errors out in python 3 with module not found error, while it works fine for Py2.
Full paths work for both.